apps/storybook/styles/storybook-styles.css

```diff
@@ -1,3 +1,5 @@
+@import url("../../../packages/features/calendars/weeklyview/styles/styles.css");
```

Comment:
Trying to learn a few things here while reviewing.
Why are we using URL import here whike we are not using it in globals.css

Reply:
Because storybook ... 

Webpack really had an issue when me and @JeroenReumkens were trying to get the base styles setup here. 

So we moved it here (techincally duplicated) but because of the way storybook imports CSS this was the only way i could find to get the styles to apply in the story